src/Countries/Maldives.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use GeniusTS\HijriDate\Hijri;
+
+class Maldives extends Country
+{
+    protected $adjustmentDays = 1; // Adjustments based on moon sighting
+
+    public function countryCode(): string
+    {
+        return 'mv';
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+        // Fixed-date holidays
+        return array_merge([
+            'New Year\'s Day' => '01-01',
+            'Labour Day' => '05-01',
+            'Independence Day' => '07-26',
+            'On the occasion of the Independence Day' => '07-27',
+            'Victory Day' => '11-03',
+            'Republic Day' => '11-11',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        // Ramadan
```

Comment:
Hi there. Could you write a test to generate the dates for 2023, 2024 and 2025 and verify if the generated results in the snapshots are correct? We don't have a proper solution for Islamic holidays yet, so I'm curious if this works.

Reply:
Hi there. 

The updated version results align with published public holidays ([Academic Calendar | Min of Education](https://www.moe.gov.mv/en/category/view/13)) of all years tested (2021, 2022, 2023, 2024, and 2025).